ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -519,19 +542,27 @@ export class AddressEmbeddedComponent implements OnDestroy, OnInit {
 
         // "Real" as opposed to "Temp". Important we return the saved address as properties are added on server
         const saveRealAddressAction$ = this.saveTrigger$.pipe(
-            withLatestFrom(this.formErrorMessages$, currentAddress$),
-            filter(([_, formErrors, addressToSave]) => {
-                return !formErrors?.length && canSaveRealAddress(addressToSave);
+            withLatestFrom(currentAddress$, this.formErrorMessages$, this.verifyFieldErrors$, ignoreEasyPostErrorsObservable),
+            filter(([_, addressToSave, formErrors, fieldErrors, ignoreEasyPost]) => {
+                return !formErrors?.filter(error => !(ignoreEasyPost && error.isEasyPostError)).length
+                    && !fieldErrors?.filter(error => !(ignoreEasyPost && error.isEasyPostError)).length
+                    && canSaveRealAddress(addressToSave);
             }),
             tap(() => busyCounter$.next(1)),
-            concatMap(([_, __, addressToSave]) => this.addressService.saveAddress(addressToSave, false)),
+            concatMap(([_, addressToSave]) => this.addressService.saveAddress(addressToSave, false)),
+            catchError((error) => {
```

Comment:
I ran into errors for saving address couple of times, so there is a sence to add catchError for that.